_docs/getting_started/usage.md

```diff
@@ -125,3 +130,12 @@ Example:
 ```
 $ lounge --version
 ```
+
+## Environment variables
+
+The Lounge is also configurable through the use of environment variables. CLI options will always have
```

Comment:
> It is also possible to configure the home path of The Lounge through the `LOUNGE_HOME` environment variable.

In fact, do we even need to make it a separate section, could it just be left under `--home`?